src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -137,6 +136,10 @@ public SwerveModuleState[] convertToModuleStates(double xTranslation, double yTr
         return kDriveKinematics.toSwerveModuleStates(chassisSpeeds);  
     }
 
+    public double getVelocity() {
+        return frontRight.getVelocity();
+    }
```

Comment:
? I don't understand this.  We have the robot velocity.  I don't think we should be looking to one module for it's velocity.  Seems problematic to me.  

Can you help me understand why we are looking at one module?

Reply:
Already fixed that was a temporary thing that I meant to delete